src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,27 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  onSelectTodo: (value: Todo | null) => void,
+  selectTodo: Todo,
+};
+
+export const TodoModal: React.FC<Props> = ({ onSelectTodo, selectTodo }) => {
+  const [selectUser, setSelectUser] = useState<User>();
+
+  useEffect(() => {
+    getUser(selectTodo.userId).then(response => setSelectUser(response));
```

Comment:
Also, u can use `async await`

```suggestion
    getUser(selectTodo.userId)
      .then(response => setSelectUser(response));
```